libs/ngrx-toolkit/src/lib/with-devtools/with-devtools.config.ts

```diff
@@ -0,0 +1,35 @@
+import { InjectionToken } from '@angular/core';
+
+export type DevtoolsConfig = {
+  logOnly: boolean;
+};
+
+export const DEFAULT_DEVTOOLS_CONFIG: DevtoolsConfig = {
+  logOnly: false,
+};
+
+export const DEVTOOLS_CONFIG = new InjectionToken<DevtoolsConfig>('DEVTOOLS_CONFIG');
+
+
+/**
+ * Provide a custom configuration for the devtools.
+ * @param config The custom configuration.
+ *
+ * @example
+ * ```ts
+ * provideStoreDevtoolsConfig({
+ *  logOnly: !isDevMode(), // Enable the log-only mode when not in production. (Default: false)
+ * });
+ *  ```
+ *
+ *  @returns The provider of the custom configuration.
+ */
+export const provideStoreDevtoolsConfig = (config: Partial<DevtoolsConfig>) => {
+  return {
```

Comment:
Do we want to make that as an environment provider so that it could only be called in the root scope or in paths?

What about override protection? I think that would also be useful.